src/applications/edu-benefits/complaint-tool/helpers.js

```diff
@@ -57,3 +60,99 @@ export function transform(formConfig, form) {
   });
 }
 
+function checkStatus(guid) {
```

Comment:
These helpers are largely based off of the [VICv2 submit helpers](https://github.com/department-of-veterans-affairs/vets-website/blob/3de62bf6a772309ebcdb422d2c669959e4704530/src/applications/vic-v2/helpers.jsx#L71-L213).